book/quick-brick/native-project-dev.md

```diff
@@ -0,0 +1,132 @@
+### Create an app for iOS/tvOS with Quick-Brick Q.A using Xcode simulators
+
+The following steps will guide you how to create a quick-brick app that will utilize the react-native quick-brick bundle from the CDN.
+
+1. Choose a Zapp app from the [Zapp](https://zapp.applicaster.com/) website, copy the app version id and save it for later stage.
+2. Go to the `ZappAppleBuilder` project folder and choose the platform you want to test:
+    * iOS -  open the `ZappiOS` folder
+    * tvOS - open the `ZappTvOS` folder
+3. Create the project by running inside the chosen directory the following command:
+
+    ```base
+    zapptool -vi <app_version_id> -pu
+    ```
+
+    In case you need to work with a local React-Native bundle, please use the following command:
+
+    ```base
+    zapptool -vi <app_version_id> -pu -rn localhost:8081
+    ```
+
+    *Notes:*
+    * The `app_version_id` is the app version id string value from earlier stage.
+    * If you prefer to run `pod update` separately, you can remove `-pu` which will execute `pod update` as part of the zapptool build process.
+
+4. Open `ZappiOS.xcworkspace` or `ZappTvOS.xcworkspace` according to earlier section.
+5. Build and run on some (iPhone / Apple TV) Simulator.
+
+### Extra information for iOS/tvOS Quick-Brick
+
+We have created extra documentations about the following issues:
+
+* [How to install a debug build on AppleTV](/quick-brick/tvOS/Environment/DebugBuildIntegration.md) - This Document will explain how to download and setup a debug version of an app on your Apple TV device using the Quickbrick SDK.
+* [How to install a release build on AppleTV](/quick-brick/tvOS/Environment/DebugBuildIntegration.md) - This Document will explain how to download and setup a release version of an app on your Apple TV device using the Quickbrick SDK.
+
+***
+
+## Android and Android T.V {#android_qb}
+
+To prepare a project for Android or Android T.V with quick-brick you need to `git clone` the [Zapp-Android](https://github.com/applicaster/Zapp-Android) repository:
+
+**Important** - The [Zapp-Android](https://github.com/applicaster/Zapp-Android) repository readme file contains information about the setup prerequisites. Please make sure you're aligned.
+
+### Create an app for Android and Android T.V with Quick-Brick Q.A using Android Emulators
+
+The following steps will guide you how to create a quick-brick app that will utilize the react-native quick-brick bundle from the CDN.
+
+1. Choose a Zapp app from the [Zapp](https://zapp.applicaster.com/) website, copy the app version id and save it for later stage.
+2. Go to the `Zapp-Android` project and run the following command:
+
+    ```base
+    rbenv local 2.5.1
+    sudo gem install bundler -v 1.17.3
+    bundle install
+    bundle exec rake prepare_workspace VERSION=<Applicastion_ID>
+    ```
+
+3. Open the Android Studio project and run the project on an Android Emulator. Note that you have two option to test the quick-brick bundle from localhost:
```

Comment:
If you want to use the react-native packager, you can choose either of these options: